examples/dataset/image_retrieval_by_text/image_retrieval_by_text/constants.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from os import path
+
+BUCKET = "kolena-public-examples"
+DATASET = "image-retrieval-by-text"
+EMBEDDING_STORAGE_DIR = path.join(path.dirname(path.realpath(__file__)), "embeddings")
+ID_FIELDS = ["caption_id"]
+
+MODELS = [
+    "kakaobrain_align-base",
+    "google_siglip-base-patch16-224",
+    "BAAI_AltCLIP",
+    "openai_clip-vit-base-patch32",
+]
```

Comment:
Any reason not to use `/` notation, so that these match directly with the `transformers` model name? e.g. `kakaobrain/align-base`

Reply:
there is no reason to not just use / notation, I will change the model names so they matches